src/pages/signup-affiliates/utils/_utils.tsx

```diff
@@ -19,6 +21,10 @@ export const customSlugify = (text: string): string => {
         .replace(/'/g, '')
         .replace(/--+/g, '-')
 }
+export const trimOnly = (text: string): string => {
+    return text.trim()
+}
```

Comment:
isn't better to use `customSlugify` which already have trim() call? Does it satisfy our needs?